SEED-platform/seed
# Store additional geocoding results

#### Any background context you want to provide?
During geocoding, there are several additional fields that are returned that can be useful such as neighborhood. These results are now stored in reserved SEED columns (e.g., `Geocoded Neighborhood`, `Geocoded City`, etc.).

#### What's this PR do?
* Fix the pre-commit file to not throw a warning about regexes
* When calling "geocode" persist off the additional data that is collected
* `get_or_create` the additional columns as needed and prevents those columns from being mappable


#### How should this be manually tested?
I had to create a new MapQuest API key since their auth systems seems to have been upgraded. I did need to provide a credit card even for the free account. I used `privacy.com` with a new credit card set with $1/year limit... just in case.

* Create new MapQuest API Key
* Put key into the settings
* Import some addresses with only address and state
* Verify that the additional geocoded data are automatically created

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
